view/src/pages/home.js

```diff
@@ -0,0 +1,194 @@
+// home.js
+
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Account from '../components/account';
+import Todo from '../components/todo';
+
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import withStyles from '@material-ui/core/styles/withStyles';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import NotesIcon from '@material-ui/icons/Notes';
+import Avatar from '@material-ui/core/Avatar';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import { authMiddleWare } from '../util/auth'
+
+const drawerWidth = 240;
+
+const styles = (theme) => ({
+	root: {
+		display: 'flex'
+	},
+	appBar: {
+		zIndex: theme.zIndex.drawer + 1
+	},
+	drawer: {
+		width: drawerWidth,
+		flexShrink: 0
+	},
+	drawerPaper: {
+		width: drawerWidth
+	},
+	content: {
+		flexGrow: 1,
+		padding: theme.spacing(3)
+	},
+	avatar: {
+		height: 110,
+		width: 100,
+		flexShrink: 0,
+		flexGrow: 0,
+		marginTop: 20
+	},
+	uiProgess: {
+		position: 'fixed',
+		zIndex: '1000',
+		height: '31px',
+		width: '31px',
+		left: '50%',
+		top: '35%'
+	},
+	toolbar: theme.mixins.toolbar
+});
+
+class home extends Component {
+	state = {
+		render: false
+	};
+
+	loadAccountPage = (event) => {
+		this.setState({ render: true });
+	};
+
+	loadTodoPage = (event) => {
+		this.setState({ render: false });
+	};
+
+	logoutHandler = (event) => {
+		localStorage.removeItem('AuthToken');
+		this.props.history.push('/login');
+	};
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			firstName: '',
+			lastName: '',
+			profilePicture: '',
+			uiLoading: true,
+			imageLoading: false
+		};
+	}
+
+	componentWillMount = () => {
```

Comment:
I think that you should use componentDidMount() instead of componentWillMount().  According to the [documentation](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount), componentWillMount() was deprecated.